src/guides/v2.3/ui_comp_guide/components/ui-wysiwyg.md

```diff
@@ -32,7 +32,22 @@ Wysiwyg-specific options:
 
 Events are dispatched by modules when certain actions are triggered. When an event is dispatched, it can pass data to any observers configured to watch that event.
 
-See [Events and observers]({{ page.baseurl }}/extension-dev-guide/events-and-observers.html) to learn more about events and how to use them.
+Use a global `varienGlobalEvents` to attach a custom event handler to WYSIWYG events. For example:
+
+```javascript
+require([
+    'jquery'
```

Comment:
I guess we don't need this dependency on `jQuery`. We should rather define `'mage/adminhtml/events'` as a dependency.

Reply:
agreed